tests/unit/amo/components/TestInstallButtonWrapper.js

```diff
@@ -504,4 +508,140 @@ describe(__filename, () => {
 
     expect(root.find('.InstallButtonWrapper-download')).toHaveLength(0);
   });
+
+  it('does not display a download link when currentVersion has no file', () => {
```

Comment:
This isn't possible anymore, see https://github.com/mozilla/addons-frontend/issues/10900